package.json

```diff
@@ -16,6 +16,7 @@
     "child-process-promise": "^2.2.0",
     "coveralls": "^2.11.6",
     "gitbook-cli": "^2.3.0",
+    "gitbook-plugin-chartjs": "^0.1.0",
```

Comment:
GitBook dependencies are installed automatically when running `gulp docs`, so we can remove this line and avoid installing GitBook plugin(s) when not generating the doc.